.github/workflows/linkchecker.yaml

```diff
@@ -0,0 +1,24 @@
+name: Run linkchecker on site
+
+on:
+  schedule:
+    - cron:  '0 0 1,15 * *'
```

Comment:
00:00 UTC is a strange time to run. Probably more efficient to run in main working hours.

Reply:
fair enough, I'll run it at 12:00